ng2-material/components/data_table/data_table_selectable_tr.ts

```diff
@@ -82,6 +82,8 @@ export class MdDataTableHeaderSelectableRow extends AbstractMdDataTableSelectabl
     this.table.onSelectableChange
       .map(event => event.allSelected)
       .subscribe(newActiveStatus => this.isActive = newActiveStatus);
+
+    this.onChange.subscribe(this.table.change.bind(this.table));
```

Comment:
I think that for this subscrpt and the one above, we should store the subscription results and unsubscribe() when the `OnDestroy` lifecycle event happens.  @Gregcop1 can you confirm that the onSelectableChange subscript should be doing this as well?